src/redux/books/api/createBook.js

```diff
@@ -0,0 +1,21 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import url from '../../../http-common';
+import { addBook } from '../books';
+
+const CREATE_BOOK = 'books/CREATE_BOOK';
+
+const createBook = createAsyncThunk(CREATE_BOOK, async (book, thunkAPI) => {
+  const response = await fetch(url, {
+    method: 'POST',
```

Comment:
- [ ] Nice work so far 👍🏿 on your project. I can see you are using the `fetch`  method for your API calls. But as per project requirement you are to use Axios for the API call instead of  fetch method